src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -300,11 +300,6 @@ medAbstractWorkspace* medToolBox::getWorkspace()
 void medToolBox::setWorkspace(medAbstractWorkspace* workspace)
 {
     d->workspace = workspace;
-    if (workspace)
-    {
-        connect(workspace->stackedViewContainers(), SIGNAL(containersSelectedChanged()), this,
-                SLOT(updateView()), Qt::UniqueConnection);
-    }
```

Comment:
We talked about this with @LoicCadour on chat and it can be left as it is in medToolBox. It won't interfere with how hide/showevent works.